### PR TITLE
add name field to ip block and iscbrf improvements v0.5.3

### PR DESCRIPTION
**Pull request by acalm**
This PR includes iscbrf improvements (akamai and dnsmadeeasy support) by @dhozac, ip blocks can now contain names!

### CODE REVIEW

**Comment by dhozac** [comment]
Looks like the tests need to be updated for name now being required. I think version should be 0.6.0 given that we're changing the required data format.

**Comment by acalm** [comment]
yea I noticed that the tests broke, I'm updating them now

File: bonk/serializers.py [hunk 1]
```diff
@@ -84,17 +87,20 @@ def create_link(self, instance):
...
     except:
         raise serializers.ValidationError("vrf=%r doesn't exist" % value)
     return value
 
+
 class IPBlockSerializer(HistorySerializerMixin):
     id = serializers.CharField(required=False)
     tags = serializers.DictField(required=False)
     vrf = serializers.IntegerField(required=True, validators=[validate_vrf])
+    name = serializers.CharField(required=True)
```
Comment (dhozac) [inline bonk/serializers.py]: I would probably put name below length, just for consistency with other things. vrf/network/length are used so much together.

File: bonk/serializers.py [hunk 2]
```diff
@@ -140,7 +150,7 @@ def get_by_ip(cls, vrf, ip, reql=False):
     @classmethod
     def filter_by_block(cls, block, reql=False):
         return cls.filter(lambda b:
-            r.ip_prefix_contains(
+                r.ip_prefix_contains(
                 r.ip_prefix(block['network'], block['length']),
```
Comment (dhozac) [inline bonk/serializers.py]: These should be indented more too, right?

File: setup.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 import re
 
 setup(name='bonk',
-      version='0.5.2',
+      version='0.5.3',
```
Comment (dhozac) [inline setup.py]: As the comment said, this should probably be 0.6.0.

**Review by dhozac — CHANGES_REQUESTED** [review]

File: bonk/serializers.py (hunk 2 above)
Comment (acalm) [reply bonk/serializers.py]: yes! however I had no intention of changing this, must've slipped, fell and landed on a TAB :wq

true story!

**Review by dhozac — APPROVED** [review]

**Pull request merged**